packages/app-store/_utils/getCalendar.ts

```diff
@@ -8,7 +8,10 @@ const log = logger.getChildLogger({ prefix: ["CalendarManager"] });
 
 export const getCalendar = (credential: CredentialPayload | null): Calendar | null => {
   if (!credential || !credential.key) return null;
-  const { type: calendarType } = credential;
+  let { type: calendarType } = credential;
+  if (calendarType === "sendgrid_other_calendar") {
+    calendarType = "sendgrid";
+  }
```

Comment:
This and the following change refer to what was discussed in #5409 with @hariombalhara regarding renaming apps with `_other_calendar` one by one.